lisa/tools/make.py

```diff
@@ -84,7 +85,7 @@ def make(
             )
 
         # yes '' answers all questions with default value.
-        self.node.execute(
+        result = self.node.execute(
```

Comment:
When is the result needed? We should handle the common checks here.

Reply:
Wanted the result to get the stdout and stderr of make command

Reply:
In my case, the caller (kernel_source_installer) reads the outputs (stdout and stderr)

Reply:
I understand some place to read it. But we need to think about why it's read, and what information is needed. Can it be parsed to structure information and then return.

Reply:
At present It is being written to a file

Reply:
Is it a requirement to upload raw build output?

Reply:
Yes, the build outputs are required for the community to debug build failures